tests/unit/dump.tcl

```diff
@@ -36,7 +36,16 @@ start_server {tags {"dump"}} {
         assert {$ttl >= 2900 && $ttl <= 3100}
         r get foo
     } {bar}
-    
+
+    test {RESTORE with ABSTTL in the past} {
+        r set foo bar
+        set encoded [r dump foo]
+        set now [clock milliseconds]
+        r restore foo [expr $now-3000] $encoded absttl REPLACE
+        catch {r debug object foo} e
```

Comment:
1. maybe turn off activeexpire (just to make sure that RESTORE never loaded the key)
2. add a comment saying that debug command doesn't trigger laxy-expire

Reply:
thanks. fixed and squashed.